mautic/mautic
# Enhance Emails UI list: Added Subject & Date sent columns

| Q | A |
| --- | --- |
| Bug fix? | n |
| New feature? | n |
| Enhancement? | y |
| Related user documentation PR URL | n |
| Related developer documentation PR URL | n |
| Issues addressed (#s or URLs) | n |
| BC breaks? | n |
| Deprecations? | n |

#### Description:
I've decided to submit all of my custom enhancements to Emails UI as PR.
This PR just adds 2 columns (with an ability to sort by them) to Emails UI list: Subject & Date sent.
In my view it greatly improves working with Emails list.
As you could see this PR also includes changes made by auto code-style fixer from #2462 .

#### Steps to test this PR:
1. Open `/s/emails`.
2. Make sure sorting by "Subject" and "Date sent" work.
